core/ustring.cpp

```diff
@@ -3758,6 +3758,35 @@ bool String::is_valid_float() const {
 	return numbers_found;
 }
 
+String String::get_final_path_from(const String &p_relative_path) const {
+
+	if (!p_relative_path.is_rel_path()) {
+
```

Comment:
Shouldn't there be a warning or something here?

Reply:
I didn't consider that possibility. I could alter the code to make it show a dev warning.